### PR TITLE
647 create the javascript view for fsk db view

### PR DESCRIPTION
**Pull request by ahmadswaid**

### CODE REVIEW

File: de.bund.bfr.knime.fsklab.nodes/plugin.xml [hunk 1]
```diff
@@ -52,6 +52,9 @@
      <node category-path="/fsklab" factory-class="de.bund.bfr.knime.fsklab.v1_9.joiner.JoinerNodeFactory" />
      <node category-path="/fsklab" factory-class="de.bund.bfr.knime.fsklab.v1_9.editor.FSKEditorJSNodeFactory" />
      
+     <!-- Please deprecate the next extension point definition before release -->
+     <node category-path="/fsklab" factory-class="de.bund.bfr.knime.fsklab.v1_9.fskdbview.FSKDBViewNodeFactory"/>
```
Comment (miguelalba) [inline de.bund.bfr.knime.fsklab.nodes/plugin.xml:56]: We should mark the node as deprecated while is developed. We can always undeprecate it to debug it or include it in a local test build.

File: de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewNodeModel.java [hunk 2]
```diff
@@ -18,13 +18,11 @@
  */
 package de.bund.bfr.knime.fsklab.v1_9.fskdbview;
 
-import java.io.File;
```
Comment (miguelalba) [inline de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewNodeModel.java:21]: This file is changed in https://github.com/SiLeBAT/FSK-Lab/pull/680. Once that PR is merged you could merge development into this branch to bring the changes.

**Review by miguelalba — CHANGES_REQUESTED** [review]
Pretty good. The only issue I notice is the node included in the plugin.xml which we do not want to appear to end users in the builds we are making now. Other than that the node model class is reviewed in #680 and I would just bring the changes from there after it is merged.

File: de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewNodeModel.java (hunk 2 above)
Comment (ahmadswaid) [reply de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewNodeModel.java:21]: Sure, waiting for PR merging :)

**Comment by ahmadswaid** [comment]
@miguelalba Could you please approve the PR if it's now OK.

**Review by miguelalba — APPROVED** [review]

**Comment by miguelalba** [comment]
Done. Please delete branch if not needed.

**Pull request merged**